notebooks/official/migration/UJ15 Vertex SDK AutoML Object Tracking.ipynb

```diff
@@ -32,17 +32,24 @@
         "# Vertex AI: Vertex AI Migration: AutoML Video Object Tracking\n",
```

Comment:
<p>combine installs of all packages into a single pip3 install</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1055/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>